src/scripts/main.js

```diff
@@ -1,3 +1,39 @@
 'use strict';
 
-// write code here
+const employeesList = document.querySelector('ul');
+
+const sortedList = sortList([...employeesList.children]);
```

Comment:
The `toSorted` method is not a standard JavaScript method for arrays. You should use the `sort` method instead to sort the array. Make sure to modify the array in place or create a new sorted array accordingly.